examples/change_attribute/change_attitude.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
```

Comment:
Thank you for your review, squilter!
I will fix this code to fit into the dronekit until May 6.
It is great to have a chance like this to contribute to the dronekit!